JitsiConference.js

```diff
@@ -595,6 +595,12 @@ JitsiConference.prototype.unlock = function () {
  * @throws NetworkError or InvalidStateError or Error if the operation fails.
  */
 JitsiConference.prototype.selectParticipant = function(participantId) {
+    // if there is no participants there is no point in sending message
+    // who is on stage, also in this moment datachannel
+    // may be not yet established
+    if (!this.participants || Object.keys(this.participants).length === 0)
+        return;
```

Comment:
But this is not correct, cause while in a call with multiple participants we need to handle switching to the local user?
And when there are multiple participants we can still have the errors in the docs.